### PR TITLE
chore: remove standalone network

### PR DESCRIPTION
**Pull request by kingpinXD**
# Description

- Removes standalone network
- Replaces assert with require to maintain consistency

Closes:https://github.com/zeta-chain/node/issues/1595
https://github.com/zeta-chain/node/issues/1696

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions 

# Checklist:

- [ ] I have added unit tests that prove my fix feature works

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -28,6 +28,7 @@
 
 * [1585](https://github.com/zeta-chain/node/pull/1585) - Updated release instructions
 * [1615](https://github.com/zeta-chain/node/pull/1615) - Add upgrade handler for version v12.1.0
+* [1694](https://github.com/zeta-chain/node/pull/1694) - remove standalone network
```
Comment (lumtis) [inline changelog.md:31]: Unreleased

**Comment by lumtis** [comment]
I think there has been a misunderstanding, we should replace `assert` with `require`

File: rpc/ethereum/pubsub/pubsub_test.go [hunk 2]
```diff
@@ -7,24 +7,24 @@ import (
 	"testing"
```
Comment (lumtis) [inline rpc/ethereum/pubsub/pubsub_test.go:7]: assert still used in this file

File: rpc/types/block_test.go [hunk 3]
```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/assert"
```
Comment (lumtis) [inline rpc/types/block_test.go:8]: same comment

File: rpc/types/events_test.go [hunk 4]
```diff
@@ -232,19 +232,19 @@ func TestParseTxResult(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			parsed, err := ParseTxResult(&tc.response, nil)
 			if tc.expTxs == nil {
-				require.Error(t, err)
+				assert.Error(t, err)
```
Comment (lumtis) [inline rpc/types/events_test.go:235]: same comment

File: server/config/config_test.go [hunk 5]
```diff
@@ -3,12 +3,12 @@ package config
...
-	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestDefaultConfig(t *testing.T) {
 	cfg := DefaultConfig()
-	require.True(t, cfg.JSONRPC.Enable)
-	require.Equal(t, cfg.JSONRPC.Address, DefaultJSONRPCAddress)
-	require.Equal(t, cfg.JSONRPC.WsAddress, DefaultJSONRPCWsAddress)
+	assert.True(t, cfg.JSONRPC.Enable)
```
Comment (lumtis) [inline server/config/config_test.go:11]: same comment

**Review by lumtis — APPROVED** [review]

**Review by lumtis — APPROVED** [review]

**Review by ws4charlie — APPROVED** [review]
looks good

**Pull request merged**